Tools/NTupleReader.cc

```diff
@@ -505,6 +506,7 @@ void* NTupleReader::getVarPtr(const std::string& var) const
     {
         e.print();
         if(reThrow_) throw;
+        else return nullptr;
```

Comment:
shouldn't be an else here really

Reply:
I know the way you setup the code you don't "need" an else, but the compiler thinks that there should be one. I think if you catch and you have "reThrow_" set to false this function will not return anything.

Reply:
my point was that literally you can remove the letters "else," I know the compiler will want the return there, but it is also fine as is